app/scripts/components/analysis/saved-analysis-control.tsx

```diff
@@ -162,9 +163,44 @@ function SavedAnalysisThumbnail(props: { aoi: FeatureCollection<Polygon> }) {
     }))
   };
 
-  const encoded = encodeURIComponent(JSON.stringify(styledFeatures));
+  let encodedGeoJson = encodeURIComponent(JSON.stringify(styledFeatures));
+  const encodedGeoJsonChars = encodedGeoJson.length;
 
-  const src = `https://api.mapbox.com/styles/v1/covid-nasa/cldac5c2c003k01oebmavw4q3/static/geojson(${encoded})/auto/480x320?access_token=${process.env.MAPBOX_TOKEN}`;
+  // If more than 8000 chars the request will fail.
+  // In this case simplify and show a bounding box.
+  if (encodedGeoJsonChars > 8000) {
```

Comment:
This is what I meant in my previous comment:
```suggestion
  const MAX_MAPBOX_API_CHARS = 8000;
  if (encodedGeoJsonChars > MAX_MAPBOX_API_CHARS) {
```